backend/src/main/java/com/google/minicrm/data/Settings.java

```diff
@@ -0,0 +1,127 @@
+package com.google.minicrm.data;
+
+import com.google.appengine.api.datastore.EmbeddedEntity;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+public final class Settings implements DatastoreObject {
+    public static String KIND_NAME = "Settings";
+
+    // used for maintaining uniqueness
+    private static long CURRENT_ID = 1;
+    private transient Key advertiserKey;
+    private long settingsId;
+
+    private String email;
+    private boolean emailNotificationsEnabled;
+    // no setter because you should get and then set instance vars
+    private NotificationsFrequency emailNotificationsFrequency;
+    private String phone;
+    private boolean phoneNotificationsEnabled;
+    // no setter because you should get and then set instance vars
+    private NotificationsFrequency phoneNotificationsFrequency;
+    private Language language;
+    private Currency currency;
+
+    // default settings created when user first created
+    public Settings(Key advertiserKey, String email) {
+        this.advertiserKey = advertiserKey;
+        this.settingsId = CURRENT_ID++;
+        this.email = email;
+        this.language = Language.ENGLISH;
+        this.currency = Currency.US_DOLLAR;
+        // defaults
+        this.emailNotificationsEnabled = false;
+        this.emailNotificationsFrequency = new NotificationsFrequency(true, false, false);
+        this.phone = "";
+        this.phoneNotificationsEnabled = false;
+        this.phoneNotificationsFrequency = new NotificationsFrequency(true, false, false);
+    }
```

Comment:
Yeah, I think an enum for notif frequencies would be much cleaner here.
Much more readable:
this.emailNotificationsEnabled = false;
this.emailNotificationsFrequency = NotifcationsFrequency.DAILY;
or even:
this.emailNotificationsFrequency = NotificationsFrequency.NEVER; and remove the boolean entirely.

etc.